dashboard/apps/dashboard/management/commands/sync.py

```diff
@@ -111,6 +112,28 @@ def sync_clients(data_dir):
             client.save()
 
 
+def sync_departments(data_dir):
+    logging.info('sync departments')
+    source = os.path.join(data_dir, 'departments.json')
+    with open(source, 'r') as sf:
+        data = json.loads(sf.read())
+    for item in data:
+        useful_data = {
+            'float_id': item['department_id'],
+            'name': item['department_name'],
+            'raw_data': item,
+        }
```

Comment:
I would suggest adding a method to `Department` to construct an instance from the JSON `item`. This could then be used in place of this code, eg: `feed_dept = Department.from_json(item)`, or even with `json.loads(sf.read(), object_hook=Department.from_json)` - this would make this code more concise, and more readable, but it would require changes to `compare` and `update`